File: 卡码/代码随想录/1/06-二叉树/Python/03-二叉树修改与构造/LeetCode_0226.py
#226. 翻转二叉树
from typing import *
from ..TreeNode import *

class Solution:
  def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
    if not root:
      return root
    root.left,root.right=root.right,root.left
    self.invertTree(root.left)
    self.invertTree(root.right)
    return root

File: 卡码/代码随想录/1/06-二叉树/Js/03-二叉树修改与构造/LeetCode_0226.js
//226. 翻转二叉树
import TreeNode from "../TreeNode.js";

const invertTree = function(root) {
  if(!root) return root;
  [root.left,root.right]=[root.right,root.left];
  invertTree(root.left);
  invertTree(root.right);
  return root;
};